doc/design/ant_xgboost.md

```diff
@@ -127,10 +127,10 @@ With the help of xgblauncher, we can launch xgboost from sqlflow AST via a light
 The `code template` roughly includes components as follows: 
 
 * `TFDataSource` that is responsible for fetching and pre-processing data via tf.feature_columns API.
-   Data will be fetched in mini-batch style by executing TF compute graph with mini-batch data feed by `sqlflow_submitter.db.db_generator`.
+   Data will be fetched in mini-batch style by executing TF compute graph with mini-batch data feed by `sqlflow_runtime.db.db_generator`.
 
 * `DBDataSource` that is responsible for writing prediction results into specific data base.
-   The writing action can be implemented via `sqlflow_submitter.db.insert_values`.
+   The writing action can be implemented via `sqlflow_runtime.db.insert_values`.
```

Comment:
Should `sqlflow_runtime` be `sqlflow.runtime`?

Reply:
The package name `sqlflow` is already used by Python client: https://github.com/sql-machine-learning/pysqlflow , I will remove the prefix `sqlflow_`.